elementary/cli/cli.py

```diff
@@ -42,7 +42,7 @@ def get_command(self, ctx, name):
     def format_help(self, ctx, formatter):
         try:
             click.echo('Loading dependencies (this might take a few seconds)')
-            config = Config()
+            config = Config(should_validate_profile=False)
```

Comment:
This is an indication that we shouldn't use the ctor for it. I suggest using config.validate_elementary_profile() in the CLI when we explicitly want to validate it. Also we can change it to return True / False and log inside the function a proper log based on the issue (file was not found or elementary profile was not found)

Reply:
The thing is that in most situations you _would_ want to validate the profile.
It's safer to remove the validation in the single place in which you wouldn't want it (the help entry),
rather than if you were to add another command in which you would have to notice that you need to call the validate method and it wouldn't happen naturally.